src/main/scala/io/viash/wrapper/BashWrapper.scala

```diff
@@ -731,9 +731,12 @@ object BashWrapper {
     val helpStrings = 
       """Viash built in Computational Requirements:
         |    ---cpus=INT
-        |        Number of CPUs to use
+        |        Number of CPUs to use. This will be passed to the component via `meta["cpus"]`.
         |    ---memory=STRING
-        |        Amount of memory to use. Examples: 4GB, 3MiB.
+        |        Amount of memory to use. Examples: 4GB, 3MiB. This will be passed to the component
+        |        via `meta["memory_b"]` and `meta["memory_kb"]`, `meta["memory_mb"]`, and so on.
```

Comment:
I agree we shouldn't list all order of magnitude fields, because there are a few more (`gb`, `tb`, `pb`), but should we maybe make mention of non-SI units, ie. `memory_kib`, `memory_mib`, `memory_gib`, `memory_tib`, `memory_pib`?

Reply:
good point!